prime-angular-frontend/src/app/modules/site-registration/site-registration-routing.module.ts

```diff
@@ -67,6 +69,18 @@ const routes: Routes = [
                 canDeactivate: [CanDeactivateFormGuard],
                 data: { title: 'Signing Authority' }
               },
+              {
+                path: SiteRoutes.ORGANIZATION_CLAIM,
+                component: OrganizationClaimPageComponent,
+                canDeactivate: [CanDeactivateFormGuard],
+                data: { title: 'Create or Claim Organization' }
```

Comment:
We don't need to indicate the organization creation in the title.  Also, we want to perform creation on OrganizationNamePage.